src/electionguard/elgamal.py

```diff
@@ -154,6 +206,71 @@ def elgamal_encrypt(
     return ElGamalCiphertext(pad, data)
 
 
+_MAC_KEY_SIZE = 256
```

Comment:
These should likely get moved to top of file.

Reply:
moved to top.